e2e_tests/tests/cluster/test_users.py

```diff
@@ -87,24 +87,9 @@ def log_in_user(credentials: authentication.Credentials, expectedStatus: int = 0
     assert child.exitstatus == expectedStatus
 
 
-def create_test_user(add_password: bool = False) -> authentication.Credentials:
-    n_username = get_random_string()
-    command = ["det", "-m", conf.make_master_url(), "user", "create", n_username]
-    subprocess.run(command, check=True)
-    # Now we activate the user.
-    child = det_spawn(["user", "activate", n_username])
-    child.expect(pexpect.EOF, timeout=EXPECT_TIMEOUT)
-    child.read()
-    child.wait()
-    child.close()
-    assert child.exitstatus == 0
-
-    password = ""
-    if add_password:
-        password = get_random_string()
-        assert change_user_password(n_username, password) == 0
-
-    return authentication.Credentials(n_username, password)
+def log_in_user(credentials: authentication.Credentials) -> None:
+    """@deprecated. use api_utils.configure_token_store directly"""
+    return api_utils.configure_token_store(credentials)
```

Comment:
This also may as well be deleted. No need to keep around deprecated things in our own testing code. I guess you can do it separately if you want to minimize the changes in this PR.